Linked List/README.markdown

```diff
@@ -218,37 +218,43 @@ It loops through the list in the same manner but this time increments a counter
 What if we wanted to find the node at a specific index in the list? With an array we can just write `array[index]` and it's an **O(1)** operation. It's a bit more involved with linked lists, but again the code follows a similar pattern:
 
 ```swift
-  public func nodeAt(_ index: Int) -> Node? {
-    if index >= 0 {
-      var node = head
-      var i = index
-      while node != nil {
-        if i == 0 { return node }
-        i -= 1
-        node = node!.next
+  public func node(atIndex index: Int) -> Node {
```

Comment:
`node(at index: Int)` would read a bit more like the other collection apis.

Reply:
Thank you for your review.
I renamed `atIndex`  to `at` by the following methods:
* node(atIndex:)
* insert(_:, atIndex:)
* remove(atIndex:)

https://github.com/raywenderlich/swift-algorithm-club/pull/662/commits/bb21f61b53dc08af0e5d5081951c683d0f57a979